pkg/cpio/types.go

```diff
@@ -33,27 +34,61 @@ type RecordFormat interface {
 	Writer(w io.Writer) RecordWriter
 }
 
-func StaticRecord(contents string, info Info) Record {
+func StaticRecord(contents []byte, info Info) Record {
 	info.FileSize = uint64(len(contents))
 	return Record{
-		bytes.NewReader([]byte(contents)),
-		info,
+		ReadCloser: NewBytesReadCloser(contents),
+		Info:       info,
 	}
 }
 
-type eofReader struct{}
+type ReadCloser struct {
+	io.Reader
+}
 
-func (eofReader) Read(p []byte) (int, error) {
-	return 0, io.EOF
+func (b ReadCloser) Close() error {
+	return nil
+}
+
+func NewBytesReadCloser(contents []byte) io.ReadCloser {
```

Comment:
I called it that initially and went back and added the closer. I don't know, if it returns a ReadCloser it seems weird that we called it a Reader.

I can change it but ...